content/en/apps/guides/hosting/requirements.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Requirements"
+linkTitle: "Requirements"
+weight: 5
+description: >
+  Requirements for hosting CHT
+relatedContent: >
+  apps/guides/hosting/self-hosting
+  apps/guides/hosting/ec2-setup-guide
+  core/guides/docker-setup
+---
+
+Hosting a CHT instance in a cloud provider like AWS or on bare-metal requires you have sufficient hardware specifications, Docker and Docker Compose installed and other infrastructure requirements met.
+
+## Hardware Requirements
+
+- 4 GiB RAM
+- 2 CPU/vCPU
+- 50 GB Hard Disk (SSD prefered)
```

Comment:
I don't see why 50GB should be listed as a minimum requirement. It should be able to run on a much lower size to start off.

Reply:
The stats I'm using here are pulled [from SRE's docs](https://github.com/medic/cht-infrastructure/tree/master/self-hosting), so I don't know what they should be.  Can you recommend a safe minimum storage amount to start with?

Reply:
For folks self hosting this, 8 GB should work as a safe starting point. As long as they are periodically monitoring it, it doesn't have to be that large.

Reply:
Perfect - I've changed it to `8 GB` and then also added a note to make sure disk usage is monitored in b043ef4